### PR TITLE
rename action classes and labels

### PR DESCRIPTION
**Pull request by devoncarew**
- rename some action classes
- renames labels of actions and key bindings for uniformity
- show an error if the keybinding is invoked and there's no running flutter app

@pq

### CODE REVIEW

File: src/io/flutter/actions/RestartFlutterAppKeyAction.java [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import com.intellij.notification.Notifications;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.jetbrains.lang.dart.ide.runner.ObservatoryConnector;
+import io.flutter.FlutterBundle;
 
 /**
- * A keystroke invoked {@link ReloadFlutterApp} action.
+ * A keystroke invoked {@link HotReloadFlutterApp} action.
```
Comment (pq) [inline src/io/flutter/actions/RestartFlutterAppKeyAction.java]: I think this should read `RestartFlutterApp`?  (My bad, sorry. 😛 )

File: src/io/flutter/run/daemon/FlutterApp.java [hunk 2]
```diff
@@ -84,12 +84,12 @@
   /**
    * Restart the app.
    */
-  void appRestart();
+  void performRestartApp();
 
   /**
    * Refresh the app.
```
Comment (pq) [inline src/io/flutter/run/daemon/FlutterApp.java]: Maybe update to say "reload"?

File: src/io/flutter/actions/RestartFlutterAppKeyAction.java (hunk 1 above)
Comment (devoncarew) [inline src/io/flutter/actions/RestartFlutterAppKeyAction.java]: done!

File: src/io/flutter/run/daemon/FlutterApp.java (hunk 2 above)
Comment (devoncarew) [inline src/io/flutter/run/daemon/FlutterApp.java]: done

**Comment by stevemessick** [comment]
Renaming that class is causing me a lot of trouble.

**Pull request merged**